### PR TITLE
Add Readability API support

### PR DESCRIPTION
**Pull request by elland**
- Not half as good as Instapaper parser, but though luck getting that source code.

### CODE REVIEW

**Comment by elland** [comment]
🚢 it?

File: iOS/Source/BookmarksController.swift [hunk 1]
```diff
@@ -4,14 +4,14 @@ import SafariServices
 class BookmarksController: SweetTableController {
     lazy var addButtonItem: UIBarButtonItem = {
         let item = UIBarButtonItem(barButtonSystemItem: .add, target: self, action: #selector(BookmarksController.addBookmark))
-        item.tintColor = .tint
+//        item.tintColor = .tint
```
Comment (3lvis) [inline iOS/Source/BookmarksController.swift]: Remove commented code

File: iOS/Source/ArticleController.swift [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import UIKit
+
+//protocol ArticleControllerDelegate: class {
```
Comment (3lvis) [inline iOS/Source/ArticleController.swift]: plz remove commented code

File: iOS/Source/BookmarksController.swift [hunk 3]
```diff
@@ -113,8 +115,19 @@ class BookmarksController: SweetTableController {
...
+            if let error = error {
+                print(error.localizedDescription)
+            }
+            if let readableArticle = readableArticle  {
+                let articleController = ArticleController(article: readableArticle, fetcher: self.fetcher)
+                self.navigationController?.pushViewController(articleController, animated: true)
+            }
+        }
+
+//        let safariController = SFSafariViewController(url: url, entersReaderIfAvailable: true)
```
Comment (3lvis) [inline iOS/Source/BookmarksController.swift]: plz remove commented code

**Comment by 3lvis** [comment]
I guess it's fine, @Elland  :)

File: iOS/Source/BookmarksController.swift (hunk 3 above)
Comment (elland) [reply iOS/Source/BookmarksController.swift]: :dolan: 🔫

**Comment by elland** [comment]
💥

**Comment by 3lvis** [comment]
Ship it

**Pull request merged**